specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/merged/unmerged.produce.flush.dynamic.hashed/client.rpt

```diff
@@ -141,6 +141,15 @@ write zilla:data.ext ${kafka:dataEx()
 write "Hello, world #A1"
 write flush
 
+write zilla:data.ext ${kafka:dataEx()
+                              .typeId(zilla:id("kafka"))
+                              .produce()
+                                  .sequence(-1)
+                                  .ackMode("LEADER_ONLY")
+                                  .build()
+                              .build()}
+write zilla:data.empty
+write flush
```

Comment:
It would be better if these script changes where not needed.
Perhaps it is only necessary to have this behavior if `reserved > 0` on `FLUSH`?
Then we can treat it as an isolated case, instead of modifying every script that sends a `FLUSH`.